src/libraries/System.Private.CoreLib/src/System/Environment.cs

```diff
@@ -11,6 +11,8 @@ namespace System
 {
     public static partial class Environment
     {
+        private static volatile int s_systemPageSize;
```

Comment:
Nit: The current style used in this file is to always have the cache statics in front of the method that does the caching.

Reply:
@jkotas done